software/beaverhabits.yml

```
@@ -0,0 +1,13 @@
+name: "Beaver Habit Tracker"
+website_url: "https://github.com/daya0576/beaverhabits"
+source_code_url: "https://github.com/daya0576/beaverhabits"
+description: "A self-hosted habit tracking app to save your precious moments in your fleeting life."
+licenses:
+  - BSD-3-Clause
+platforms:
+  - Python
+  - Docker
+tags:
+  - Time Tracking
+  - Miscellaneous
```

By having a category, it is inherently not misc so this can be removed. 

```suggestion
tags:
  - Time Tracking
```